revert/opnsense-revert.8

```diff
@@ -49,10 +50,20 @@ Ignore the signature verification result.
 Select the
 .Ar release
 of the package to be installed.
-Note that the release is not the individual package version.
+Note that the release is not the individual package version,
+but the version found in said OPNsense release.
 .El
 .Sh EXIT STATUS
 .Ex -std
+.Sh USAGE EXAMPLES
+Revert unbound to the version found in OPNsense 17.7.1:
+opnsense-revert -r 17.7.1 unbound
```

Comment:
Use this for shell blocks according to OpenBSD:

.Pp
.Dl # opnsense-revert -r 17.7.1 unbound
.Pp